unvoid-io/js-client
# #326 CreatablePlaybook and UpdablePlaybook is missing author property

Adresses: [#326](https://github.com/gravwell/js-client/issues/326)

How to test it ?

On the gravgui front-end, go to `package.json`, then inside the dependencies, look for `gravwell`, then change the package number release, to the current path of this directory in your local machine, in my case it is:
![Captura de tela de 2022-06-08 16-47-38](https://user-images.githubusercontent.com/69917459/172704611-b490d9a6-88d3-466e-abae-fc4662e4d859.png)

After that go to the path of this directory in you machine then run : `npm run build`

Come back to gragui front-end, then run: `npm install ../js_client_unvoid`
** Note that the path after the word `install` is the same that you have in your packacke.json -> dependencies -> gravwell

Run `npm run serve`

Disable the cache

Finally, see if the changes are working as expected.
